[DOCS] Rework conceptual info for ILM. (#52181)
elastic/elasticsearch: docs/reference/ilm/actions/ilm-readonly.asciidoc

[[ilm-readonly]]
=== Read-Only

Phases allowed: warm.

> This action will set the index to be read-only
> (see: <<dynamic-index-settings,index.blocks.write>>)

Reviewer: ```suggestion
Sets an index as read-only. See <<dynamic-index-settings,index.blocks.write>>.
```

> This action does not have any options associated with it.

Reviewer: Not sure if this is needed. Either way, it's not included in the freeze action. I'd be consistent either way.

> [source,console]

Reviewer: Examples heading needed.
